src/Data/LLVM/BitCode/Parse.hs

```diff
@@ -833,14 +845,15 @@ failWithContext' msg =
        , errContext = envContext env
        }
 
-liftFinalize :: Finalize a -> Parse a
-liftFinalize (Finalize m) =
+liftFinalize :: FuncSymTabs -> Finalize a -> Parse a
```

Comment:
Please leave some comments describing this `FuncSymTabs`. Under what circumstances should this be passed an empty `Map` versus a non-empty one?

Reply:
I've added some general Haddocks.

Passing an empty map is only done in `Function.hs` in two places. Both of those places I find a little suspicious. Resolving those suspicions would take an hour or two of investigation, code changes, and testing, whereas passing an empty map preserves the functionality that was originally there.  I'd rather not add documentation that asserts something I'm not confident about because I wouldn't want to mislead future development efforts.